ignitionrobotics/ign-gazebo
# Add view control gui plugin and support orthographic view

## Summary

depends on https://github.com/ignitionrobotics/ign-rendering/pull/322

* Updated view controller in Scene3d to support both orbit and orthographic view controller
* Added View Control gui plugin that makes service request to Scene3D to toggle between the two view controllers

I can port the changes here to ign-gui if it helps in merging the two Scene3D classes.

## To Test

Run ign-gazebo, open the GUI plugin menu (3 dots on top right) and select `View Control` plugin. Choose `Orthographic` from the drop down list to see the new view controller.

![ortho_view_controller_gazebo](https://user-images.githubusercontent.com/4000684/118082246-aec8be80-b371-11eb-996d-46199b2b902e.png)


## Checklist

**Note to maintainers**: Remember to use **Squash-Merge**
